forms/subscribedialog.ui

```diff
@@ -19,512 +19,852 @@
-       </item>
-       <item>
-        <property name="text">
-         <string>EXCLUSIVE_OWNERSHIP</string>
-        </property>
-       </item>
-      </widget>
-     </item>
-     <item row="9" column="1">
-      <widget class="QCheckBox" name="checkBox_timeBasedFilter">
-       <property name="font">
-        <font>
-         <pointsize>13</pointsize>
-        </font>
+        <string>ORANGE</string>
        </property>
+      </item>
+      <item>
        <property name="text">
-        <string>Time Based (ms):</string>
-       </property>
-      </widget>
-     </item>
-     <item row="4" column="2">
-      <widget class="QSpinBox" name="spin_ownershipStrength">
-       <property name="enabled">
-        <bool>false</bool>
-       </property>
-       <property name="minimum">
-        <number>0</number>
-       </property>
-       <property name="maximum">
-        <number>50</number>
-       </property>
-       <property name="value">
-        <number>0</number>
-       </property>
-      </widget>
-     </item>
-    </layout>
-   </item>
-   <item>
-    <widget class="QDialogButtonBox" name="buttonBox">
-     <property name="orientation">
-      <enum>Qt::Horizontal</enum>
-     </property>
-     <property name="standardButtons">
-      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
-     </property>
-    </widget>
-   </item>
-  </layout>
+        <string>MAGENTA</string>
+       </property>
+      </item>
+     </widget>
+    </item>
+    <item row="1" column="2">
+     <widget class="QSpinBox" name="spin_Size">
+      <property name="enabled">
+       <bool>false</bool>
+      </property>
+      <property name="minimum">
+       <number>1</number>
+      </property>
+      <property name="value">
+       <number>30</number>
+      </property>
+     </widget>
+    </item>
+    <item row="1" column="4">
+     <widget class="QCheckBox" name="checkBox_B">
+      <property name="text">
+       <string>B</string>
+      </property>
+     </widget>
+    </item>
+    <item row="0" column="0">
+     <widget class="QLabel" name="label_3">
+      <property name="font">
+       <font>
+        <pointsize>13</pointsize>
+       </font>
+      </property>
+      <property name="text">
+       <string>Shape</string>
+      </property>
+      <property name="textFormat">
+       <enum>Qt::AutoText</enum>
+      </property>
+     </widget>
+    </item>
+   </layout>
+  </widget>
  </widget>
+ <tabstops>
+  <tabstop>combo_Shape</tabstop>
+  <tabstop>combo_Color</tabstop>
+  <tabstop>spin_Size</tabstop>
+  <tabstop>checkBox_A</tabstop>
+  <tabstop>checkBox_B</tabstop>
+  <tabstop>checkBox_C</tabstop>
+  <tabstop>checkBox_D</tabstop>
+  <tabstop>checkBox_Asterisk</tabstop>
+  <tabstop>spin_HistoryQos</tabstop>
+  <tabstop>checkBox_reliable</tabstop>
+  <tabstop>comboBox_durability</tabstop>
+  <tabstop>comboBox_liveliness</tabstop>
+  <tabstop>lineEdit_leaseDuration</tabstop>
+  <tabstop>lineEdit_announcementPeriod</tabstop>
```

Comment:
The announcement period only applies to the publisher, it shouldn't be part of the subscriber configuration, can we remove it?